src/fragment-components/a-code-snippet.tsx

```diff
@@ -109,6 +124,7 @@ export const componentInfo: ComponentInfo = {
 			imports: ['CodeSnippetModule'],
 			code: ({ json }) => {
 				return `<ibm-code-snippet
+					${json.light ? `[theme]="light"` : `[theme]="dark"`}
```

Comment:
We have similar code elsewhere, use the same pattern. (also, this feels like it would fail lint because of template strings)